google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -192,7 +192,7 @@ clouddebug.warninglabel=Warning\:
 clouddebug.attachingtext=Attaching...
 clouddebug.stopandcontinue=Stop IDE Debugging.
 clouddebug.exitdebug=Exit debug mode and continue the debug session later.
-clouddebug.message.title=Cloud Debugger
+clouddebug.message.title=Stackdriver Debugger
```

Comment:
Should we remove this one and update references to "clouddebug.stackdriver.debugger" above?